### PR TITLE
Retrofix

### PR DESCRIPTION
**Pull request by hellfire-bloodyb**
Fix crash with opening games

### CODE REVIEW

File: vita3k/kernel/src/thread/sync_primitives.cpp [hunk 1]
```diff
@@ -514,12 +513,12 @@ int condvar_wait(KernelState &kernel, const char *export_name, SceUID thread_id,
...
-	if (auto error = handle_timeout(thread, thread_lock, condition_variable_lock, *condvar, data, export_name, timeout))
-		return error;
+    if (auto error = handle_timeout(thread, thread_lock, condition_variable_lock, *condvar, data, export_name, timeout))
+        return error;
 
-	thread_lock.unlock();
+    thread_lock.unlock();
 
-	return mutex_lock_impl(kernel, export_name, thread_id, 1, condvar->associated_mutex, weight, timeout, false);
+    return mutex_lock_impl(kernel, export_name, thread_id, 1, condvar->associated_mutex, weight, timeout, false);
```
Comment (Zangetsu38) [inline vita3k/kernel/src/thread/sync_primitives.cpp:521]: What is this change ?

**Comment by hellfire-bloodyb** [comment]
Vita3k crashes when opening a game on my pc

**Comment by Zangetsu38** [comment]
> Vita3k crashes when opening a game on my pc
No i said, what is this change one line i have indicate ? Why modify format ?

File: vita3k/kernel/src/thread/sync_primitives.cpp (hunk 1 above)
Comment (ichisadashioko) [reply vita3k/kernel/src/thread/sync_primitives.cpp:521]: white spaces and tabs by `clang-format`

**Comment by hellfire-bloodyb** [comment]
For low end pc

**Comment by ichisadashioko** [comment]
@hellfire-bloodyb Did this fix your problem with booting game?

**Comment by hellfire-bloodyb** [comment]
Nope

**Comment by ichisadashioko** [comment]
@hellfire-bloodyb So why do you create this PR?

**Comment by hellfire-bloodyb** [comment]
To see if they will fix it

**Comment by ichisadashioko** [comment]
@hellfire-bloodyb [Here is the discord link](https://discord.gg/MaWhJVH) in case you miss it at the README. Please join the discord so that we can help you with the PR.

**Comment by hellfire-bloodyb** [comment]
Thanks

**Comment by 1whatleytay** [comment]
PRs are for submitting code to contribute to a project. It seems you're talking about an issue? Feel free to make an issue or submit some of your own code.

**Pull request closed without merging**